README.md

```diff
@@ -12,7 +12,6 @@ This action enables the integration of PagerDuty into StackStorm. It is capable
 Copy the example configuration in [pagerduty.yaml.example](./pagerduty.yaml.example)
 to `/opt/stackstorm/configs/pagerduty.yaml` and edit as required.
 
-* `subdomain:` name of subdomain
 * `api_key:` API-KEY
 * `service_api:` SERVICE-API
 * `debug:` optional debug flag. Set to True for additional logging
```

Comment:
It's something we need to do more consistently in all of our READMEs. I probably should have added when I updated those 100 or so packs to use config.schema.